pkg/multiboot/trampoline.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 the u-root Authors. All rights reserved
```

Comment:
Comment on pkg/multiboot/trampoline, could you include an .S file instead of a binary? That way the origin and review of it is way clearer.